### DISCUSSION TITLE
Exception in thread "main" java.lang.NullPointerException

### DISCUSSION THREAD

**Discussion by VladyslavZagorodnyi** [qa-guru/allure-notifications#111] [Q&A] [answered]
Hello, I have NullPointerException 
![image](https://user-images.githubusercontent.com/29700638/178538457-02d344f3-2fe8-46da-8063-f4f1a0efc16c.png)
but, as you can see I indicated path to config.json
![image](https://user-images.githubusercontent.com/29700638/178537493-bd01bd59-3863-4a9e-a20b-ffafc43d108b.png)
also you can see that jar file and config.json are located at root directory
![image](https://user-images.githubusercontent.com/29700638/178538298-1ba1f062-5377-41ca-aecf-0bf7259d9b4a.png)

Pls, suggest what's wrong here

**Comment by svasenkov** [comment]
Show content of config.json, please

**Reply by VladyslavZagorodnyi** [reply]
@svasenkov  here is content of config.json
![image](https://user-images.githubusercontent.com/29700638/178674126-c501b552-7ea6-4acf-b02a-f21f156336cf.png)

**Comment by valfirst** [comment, marked as answer]
the argument name is invalid, it should be `-DconfigFile=...`

**Reply by VladyslavZagorodnyi** [reply]
Thanks
